lib/prosperworks/pipeline_stage.rb

```diff
@@ -0,0 +1,14 @@
+module ProsperWorks
+  class PipelineStage < BaseEntity
```

Comment:
Same here, should inherit from `Base`. `BaseEntity` is meant for the 6 main entity types (lead, person, project, etc.)